cmd/rm-main.go

```diff
@@ -210,7 +210,7 @@ func checkRmSyntax(ctx context.Context, cliCtx *cli.Context, encKeyDB map[string
 		url = filepath.ToSlash(filepath.Clean(url))
 		// namespace removal applies only for non FS. So filter out if passed url represents a directory
 		dir := isAliasURLDir(ctx, url, encKeyDB, time.Time{})
-		if !dir {
+		if dir {
```

Comment:
Can you use isNamespaceRemoval() here ?

Reply:
In the code we check if its a directory ,  then find the path to decide if its a name space removal or not. So `isNamespaceRemoval` cant be used there.

Reply:
> In the code we check if its a directory , then find the path to decide if its a name space removal or not. So `isNamespaceRemoval` cant be used there.

I didn't understand. What I am saying is it L210->L216 seems to be checking if this is a namespace removal or not. Which is what isNamespaceRemoval() is doing as well. So my suggestion is to remove L210->L216 line and replace it with isNamespaceRemoval()